app/lib/actions/pulp/repository/create_in_plan.rb

```diff
@@ -7,11 +7,15 @@ class CreateInPlan < Create
         def plan(input)
           plan_self(input)
           pulp_extensions.repository.create_with_importer_and_distributors(input[:pulp_id],
-                                                                                      importer,
-                                                                                      distributors,
-                                                                                      display_name: input[:name])
+                                                                           importer,
+                                                                           distributors,
+                                                                           display_name: input[:name])
         rescue => e
-          raise error_message(e.http_body) || e
+          begin
+            e = error_message(e.http_body) || e
+          rescue
+          end
+          raise e
```

Comment:
Personally i like the second one. ;)
Actually i would prefer to only catch and rethrow HTML-errors. I just didn't know which ones.
That way, the original backtrace could be preserved.